### ISSUE TITLE
VideoLearner inside utils_cv.action_recognition.model [ASK]

### ISSUE THREAD

**Issue by ubaidseth** [microsoft/computervision-recipes#575]
### Description
I'm looking at the Action Recognition model by loading a video file. It seems VideoLearner is using predict_video. Is there a way to use this to split or trim a video based off the classes? In other words, exporting/saving multiple video files from one based off the class presented?

**Comment by PatrickBue** [comment]
Not 100% what you mean. In its current version we do not support trimming of videos (e.g. ffmpeg could maybe be used for that)

**Comment by ubaidseth** [comment]
Sorry, let me rephrase. 

Currently, using Jupyter, my last line of code is 
```
learner.predict_video(
    video,
    LABELS,
    averaging_size=AVERAGING_SIZE,
    score_threshold=SCORE_THRESHOLD,
    target_labels=TARGET_LABELS,
)
```

How do I loop over all the prediction labels so I can cut out subclips?

**Comment by PatrickBue** [comment]
Fyi @jiata
